testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js

```diff
@@ -153,7 +153,10 @@ Class('NX.TestClass', {
         },
         { type: 'admin[TAB]', target: this.cq1('#username') },
         { type: 'admin123', target: this.cq1('#password') },
-        { type: '[ENTER]' }
+        { type: '[ENTER]' },
+        function(next) {
+          this.waitForAnimations(next);
+        }
```

Comment:
FTR I will be going through and removing any extraneous calls in existing code shortly